server.js

```diff
@@ -47,6 +47,20 @@ app.post('/accounts', (req, res) => {
 	}
 })
 
+// DELETE /accounts/:name
+app.delete('/accounts/:name', (req, res) => {
+	let name = req.params.name;
+	let accounts = accountsDB.filter((account) => account.name !== name);
+	if (accounts.length != accountsDB.length) {
+		res.send('Account deleted');
```

Comment:
This is not bad at all, but it seems that you are not deleting the user for the database.
Please fix this and it will be ready to merge 👍